LICENSE

```diff
@@ -1,7 +1,7 @@
 Pike, with the exception of pykerb and certain build scripts
-(see below) is licensed under the following terms:
+(see below), is licensed under Simplified BSD License as follows:
 
-  Copyright (c) 2013, EMC Corporation
+  Copyright © 2013-2020, Dell Inc. or its subsidiaries.
```

Comment:
I thought if we give a range we need to update year on year?

Reply:
i think it's just an update if it changes... but IANAL

Reply:
Anyways we can figure it out next year. I assume when it's next touched since a new file this year has only this year in its copyright info.

Reply:
Good question...